scripts/Create-StorageContainer.ps1

```diff
@@ -132,40 +168,90 @@ foreach($storageContainerName in $storageContainerList)
         {
             $response = az storage container list --account-name $StorageAccountName --account-key $StorageAccountKey --prefix $storageContainerName -o json
         }
+    }
+
+    $status = (-Not ( ($response | jq --raw-output ".[] | select(.name==\`"$storageContainerName\`") | .name") -ine $storageContainerName) )
+    if ($status)
+    {
+        Log-Message "'$storageContainerName' already exists."
+    }
+    else
+    {
+        Log-Message "'$storageContainerName' doesn't exist."
+    }
+
+    return $status
+}
+
+function Set-ContainersPermissions
```

Comment:
- [ ] This function may need a re-work. It looks very similar to New-Containers. Instead of doing a full iteration, etc, would it be better to let this function deal with setting permission for a single container and call it (conditionally) from New-Containers instead of as a separate call that possibly takes additional iteration?